src/raster/r.surf.volcano/r.surf.volcano.py

```diff
@@ -0,0 +1,385 @@
+#
+# MODULE:       r.surf.volcano
+#
+# AUTHOR:       M. Hamish Bowman, Dept. of Geology, University of Otago
+#                        Dunedin, New Zealand
+#               Ported to Python from GRASS 6 addons shell script 8/2024
+#
+# PURPOSE:      Create an artificial surface resembling a seamount or cone volcano
+#
+# COPYRIGHT:    (c) 2009-2024 Hamish Bowman, and the GRASS Development Team
+#               This program is free software under the GNU General Public
+#               License (>=v2). Read the file COPYING that comes with GRASS
+#               for details.
+#
+#                This program is distributed in the hope that it will be useful,
+#                but WITHOUT ANY WARRANTY; without even the implied warranty of
+#                MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#                GNU General Public License for more details.
+#
+#############################################################################
+
+# %Module
+# % description: Creates an artificial surface resembling a seamount or cone volcano.
+# % keyword: raster
+# %End
+# %Option G_OPT_R_OUTPUT
+# %End
+# %Option
+# % key: peak
+# % type: double
+# % required: no
+# % description: Height of cone
+# % answer: 1000.0
+# %End
+# %Option
+# % key: crater
+# % type: double
+# % required: no
+# % label: Depth of crater below the cone
+# % description: A larger (deeper) value here also means a wider crater.
+# % answer: 0.0
+# %End
+# %Option
+# % key: method
+# % type: string
+# % required: no
+# % description: Mathematical function for creating the mountain
+# % answer: polynomial
+# % options: polynomial,gaussian,lorentzian,logarithmic,exponential
+# % descriptions: polynomial;1/distance^n;gaussian;Gaussian function;lorentzian;Cauchy-Lorentz distribution;logarithmic;Logarithmic decay;exponential;Exponential decay
+# %End
+# %Option
+# % key: friction
+# % type: integer
+# % required: no
+# % options: 1-25
+# % label: Polynomial friction of distance, (the 'n' in 1/d^n)
+# % description: Higher values generate steeper slopes. (only used with the polynomial method)
+# % answer: 6
+# %End
+# # FIXME: ok, it isn't really kurtosis but it's similar and I couldn't
+# #        think of a better name.
+# %Option
+# % key: kurtosis
+# % type: double
+# % required: no
+# % label: Slope steepness (used with all methods except polynomial)
+# % description: For Gaussian: nearer to 0 is flatter, higher values generate steeper slopes. For Lorentzian, logarithmic, and exponential the opposite is true.
+# % answer: 1.0
+# %End
+# %Option
+# % key: sigma
+# % type: double
+# % required: no
+# % label: Surface roughness factor
+# % description: Nearer to 0 is smoother, higher values make a rougher surface.
+# % answer: 1.0
+# %End
+# %Flag
+# % key: r
+# % description: Roughen the surface
+# %End
+
+import os
+import sys
+from math import pi
+from grass.script import core as gc
+from grass.script import raster as gr
```

Comment:
```suggestion
import grass.script as gs
```

is what is usually used for simplicity.